system-test/spanner.ts

```diff
@@ -1051,13 +1070,23 @@ describe('Spanner', () => {
     });
 
     after(async () => {
-      await restoreDatabase.delete();
```

Comment:
Would doing this here work:?

```ts
if (emulatorEnabled) {
  this.skip();
}
```

Reply:
This doesn't work. I tried this at the beginning.

Reply:
I got the following error: 

```bash
     Error: `this.skip` forbidden
```

Reply:
I did some research. The [documentation](https://mochajs.org/#inclusive-tests) says to just do a `return` instead of a `skip`. I updated the code.

Reply:
Good catch. Thanks for making this more elegant.